eip155/caip10.md

```diff
@@ -56,6 +74,10 @@ eip155:137:0x0495766CD136138FC492DD499B8DC87A92D6685B
 
 ```
 
+## Security Concerns
+
+We should have users sign a message to prove they have an EOA (Greg to fill out more completely)
+
 ## References
 
 - [EIP155][]: Ethereum Improvement Proposal specifying generation and validation of ChainIDs
```

Comment:
```suggestion
- [EIP-155][]: Ethereum Improvement Proposal specifying generation and validation of ChainIDs
- [EIP-2294][]: Explicit Bound to ChainID Size
```

Reply:
apologies for the style-guide chaos around EIPXXX versus EIP-XXX

Reply:
no worries. Thanks for the suggestion